mage_ai/tests/data_cleaner/statistics/test_calculator.py

```diff
@@ -37,26 +40,26 @@ def test_calculate_statistics_overview(self):
         self.assertEqual(data['age/median'], 1)
         self.assertEqual(data['age/min'], 1)
         self.assertEqual(data['age/sum'], 8)
-        self.assertEqual(data['age/null_value_rate'], 2/7)
+        self.assertEqual(data['age/null_value_rate'], 2 / 7)
         self.assertEqual(data['age/completeness'], 1 - data['age/null_value_rate'])
         self.assertEqual(data['age/quality'], 'Bad')
 
         self.assertEqual(data['country/count'], 5)
         self.assertEqual(data['country/count_distinct'], 3)
         self.assertEqual(data['country/mode'], 'US')
-        self.assertEqual(data['country/null_value_rate'], 2/7)
+        self.assertEqual(data['country/null_value_rate'], 2 / 7)
         self.assertEqual(data['country/completeness'], 1 - data['country/null_value_rate'])
         self.assertEqual(data['country/quality'], 'Bad')
 
-        self.assertEqual(data['date_joined/count'], 6)
+        self.assertEqual(data['date_joined/count'], 5)
```

Comment:
hmm, the count should still be 6 although one datetime is invalid. there're 6 values in the column.

Reply:
Count is 5 when the statistics calculator is asked to clean the underlying data (the `899-07-01 00:00:00+00:00` entry is out of bounds and gets converted to null). The test was run with is_clean = False, meaning that this conversion is done automatically. I can duplicate the test result to run the same calculation without cleaning the data.

Reply:
Could you remind me when you removed datetime conversion in last PR and add it back in this PR? https://github.com/mage-ai/mage-ai/pull/127/files#diff-fa21d6676bb07fd7cdc580cb751edc1f815f2ec14adb839d3fc5d72401dfcd2dR35-R36

Reply:
ok, i see. we might need to pass both cleaned and uncleaned df to stats calculator as well. some stats need to be calculated on uncleaned dataframe, like count(), and [invalid_value_count](https://github.com/mage-ai/mage-ai/blob/master/mage_ai/data_cleaner/statistics/calculator.py#L212)

Reply:
This feature of the cleaned and uncleaned is currently in todo phase, should we shift towards making that a main priority?

Reply:
Let's do that after fixing the P0 bugs.